src/components/ReportActionItem/TransactionPreview/TransactionPreviewContent.tsx

```diff
@@ -0,0 +1,288 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import Button from '@components/Button';
+import Icon from '@components/Icon';
+import {DotIndicator, Folder, Tag} from '@components/Icon/Expensicons';
+import MoneyRequestSkeletonView from '@components/MoneyRequestSkeletonView';
+import MultipleAvatars from '@components/MultipleAvatars';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import PressableWithoutFeedback from '@components/Pressable/PressableWithoutFeedback';
+import ReportActionItemImages from '@components/ReportActionItem/ReportActionItemImages';
+import UserInfoCellsWithArrow from '@components/SelectionList/Search/UserInfoCellsWithArrow';
+import Text from '@components/Text';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import ControlSelection from '@libs/ControlSelection';
+import {convertToDisplayString} from '@libs/CurrencyUtils';
+import {canUseTouchScreen} from '@libs/DeviceCapabilities';
+import {getCleanedTagName} from '@libs/PolicyUtils';
+import variables from '@styles/variables';
+import type {TransactionPreviewUIProps} from './types';
+
+function TransactionPreviewContent({
+    isDeleted,
+    isScanning,
+    isWhisper,
+    isHovered,
+    isSettled,
+    isBillSplit,
+    isApproved,
+    isSettlementOrApprovalPartial,
+    isReviewDuplicateTransactionPage,
+    shouldShowSkeleton,
+    shouldShowRBR,
+    shouldDisableOnPress,
+    shouldShowKeepButton,
+    shouldShowDescription,
+    shouldShowMerchant,
+    shouldShowCategory,
+    shouldShowTag,
+    displayAmount,
+    category,
+    showCashOrCard,
+    tag,
+    requestCurrency,
+    merchantOrDescription,
+    previewHeaderText,
+    requestAmount,
+    splitShare,
+    receiptImages,
+    sortedParticipantAvatars,
+    containerStyles,
+    walletTermsErrors,
+    pendingAction,
+    showContextMenu,
+    offlineWithFeedbackOnClose,
+    translate,
+    navigateToReviewFields,
+    onPreviewPressed,
+    RBRmessage,
+    from,
+    to,
+    isIOU,
+}: TransactionPreviewUIProps) {
+    const theme = useTheme();
+    const styles = useThemeStyles();
+
+    const themeStyles = useMemo(
+        () => [
+            {
+                backgroundColor: theme.cardBG,
+            },
+        ],
+        [theme.cardBG],
+    );
+
+    const shouldShowCategoryOrTag = shouldShowCategory || shouldShowTag;
+
+    const childContainer = (
```

Comment:
```suggestion
    const transactionContent = (
```
or something similar to this?

this goes _inside_ some code, so container name doesn't exactly fit.
And sure these are children, but so are many other components - I think children is just too generic of a name.